sdks/RepoToolset/tools/Version.props

```diff
@@ -36,8 +36,18 @@
   <Choose>
     <When Condition="'$(OfficialBuild)' == 'true'">
       <PropertyGroup>
-        <_BuildNumberFiveDigitDateStamp>$([MSBuild]::Subtract($(BUILD_BUILDNUMBER.Split('.')[0].Substring(3).Trim()), 8800))</_BuildNumberFiveDigitDateStamp>
-        <_BuildNumberBuildOfTheDayPadded>$(BUILD_BUILDNUMBER.Split('.')[1].PadLeft(2,'0'))</_BuildNumberBuildOfTheDayPadded>
+        <!-- Split the build parts out from the BuildNumber which is given to us by MicroBuild in the format of yyyymmdd.nn
```

Comment:
Perhaps it's time to define a task and write this logic in C# instead of this mess of msbuild calls...

Reply:
Maybe (and to update the logic so that it doesn't require updating every year or two, since this new logic will still fail at some point in the future). But that is probably a future change since everything is on the floor right now.

Reply:
@tannergooding How will the new logic fail (other than actually running out of numbers which obviously requires you to restart)

Reply:
Its still a general issue and we have to ensure that resetting coincides with a increment of one of the other version numbers.